api/controllers/storyText.js

```diff
@@ -0,0 +1,16 @@
+const generateStory = require("../clients/GPTclient");
+const GPT_prompt_gen = require("../prompts/GPTPromptGen")
+
+const StoryController = {
+  Index: async (req, res) => {
+    try {
+      const prompts = GPT_prompt_gen(req.body.userinput)
```

Comment:
would probably need to update the req.body.userinput to correspond with the variable naming coming from the front end